app/assets/javascripts/app/views/help_view.js

```diff
@@ -187,8 +197,27 @@ app.views.Help = app.views.StaticContentView.extend({
     e.preventDefault();
   },
 
+  chat: function(e){
+    this.renderStaticSection("chat", "faq_chat", this.CHAT_SUBS);
+    this.menuClicked(e);
+
+    e.preventDefault();
+  },
+
   linkHtml: function(url, text) {
     return "<a href=\"" + url + "\" target=\"_blank\">" + text + "</a>";
+  },
+
+  chatEnabled: function(){
+    return gon.chatEnabled;
+  },
+
+  getChatIcons: function(){
+    return '<div class="help-chat-icons"> \
+              <i class="entypo lock-open"></i> \
+              <i class="entypo chat"></i> \
+              <i class="entypo trash"></i> \
+            </div>';
```

Comment:
Expected '<div class="help-chat-icons">               <i class="entypo lock-open"></i>               <i class="entypo chat"></i>               <i class="entypo trash"></i>             </div>' to have an indentation at 5 instead at 1.<br>Expected an assignment or function call and instead saw an expression.